plugin/audit_null/audit_null.cc

```diff
@@ -130,6 +132,11 @@ static char *g_record_buffer;
 
 #undef AUDIT_NULL_VAR
 
+static const constexpr size_t event_response_buffer_len = 1000;
```

Comment:
Should `event_response_buffer_len` and `generic_event_response` definitions be also wrapped in `#ifndef DBUG_OFF`?